sources/packages/web/src/store/modules/student/getters.ts

```diff
@@ -21,4 +21,7 @@ export const getters: GetterTree<StudentState, RootState> = {
         restriction.type === RestrictionNotificationType.Error,
     );
   },
+  firstName(state: StudentState): string {
```

Comment:
Do we need a specific getter for the `firstName`?
I believe that we can have a composable to return the `StudentState`. In this way, we can keep exposing properties and the composable will have whatever we add to the `StudentState`.